ui/apps/ui/src/app/collections/data/utils.ts

```diff
@@ -70,21 +71,65 @@ export const formatPublicationDate = (
   return publication_date ? moment(publication_date).format('YYYY') : '';
 };
 
-export const resolveDoiLink = (rawDoi: string): string | undefined => {
+const resolveDoiLink = (rawDoi: string): string | undefined => {
   const doiRegex = new RegExp('^10\\.\\d{4,9}/[-._;()/:A-Za-z0-9]+$');
   return doiRegex.test(rawDoi) ? `https://doi.org/${rawDoi}` : undefined;
 };
 
-export const constructDoiTag = (
-  doiArr: string[] | undefined
+const resolveHandleLink = (rawHandle: string): string | undefined => {
+  const handleRegex = new RegExp('^\\S+/\\S+$');
+  return handleRegex.test(rawHandle)
+    ? `https://hdl.handle.net/${rawHandle}`
```

Comment:
Current PID:
- DOI: `https://doi.org/${rawDoi}`

This PR added:
- HANDLE: `https://hdl.handle.net/${rawHandle}`
- PMC: `https://europepmc.org/article/PMC/${urlPart}`
- PMID: `https://pubmed.ncbi.nlm.nih.gov/${rawPmid}`

Total statistics for PIDs:
```
has_doi     3947182
no pid      1093032
has_handle   169408
has_pmid      53261
has_pmc       23839
has_arXiv     12011
has_w3id       3051
has_pdb           2
```
- Shouldnt we also add arXiv and w3id PID? @agpul @wujuu 

@Marcelinna @agpul 
The PIDs that have been added, need to be tested on this PR. Furthermore, it is necessary to confirm that those links are proper for each PID. Maybe Anna Wałek could confirm this?

Reply:
Based on @wiktorflorian 's research - those as the correct linka for these pids. We basically haven't found any counter examples where they wouldn't work